modules/cxf/src/main/java/org/mule/module/cxf/WSProxyService.java

```diff
@@ -71,7 +71,7 @@
     private static final String WSDL_PARAM_2 = "&wsdl";
 
     /** This is an internal semaphore, not a property */
-    private boolean lazyInit = true;
+    private boolean lazyInit = false;
```

Comment:
not true. Look at the initialise() and onCall() methods. It might be true that the class can be refactored to not need it. However, the class is deprecated and such change out of scope